index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Document</title>
+</head>
+<body>
+  <p class="header">
+    Hello Microverse!
+  </p>
```

Comment:
- [ ] OPTIONAL: I suggest you use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp), because it's easier for both people and machines to read and understand it.